app/src/main/java/com/navapp/data/DefaultStoreDao.java

```diff
@@ -17,8 +19,18 @@
 
     @Insert
     void insert(DefaultStoreModel store);
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
So... What is this conflict and why are you ignoring it?

Reply:
`CONFLICT` refers to a situation where an `insert` cannot be completed because it conflicts with an existing row. In the `defaults` table, this will occur if there is already a row for the same table. In such a case, we don't want an exception from this method, just a simple failure which returns `-1` (unlike the other `insert`). That's what `IGNORE` does.